.travis.yml

```diff
@@ -1,29 +1,71 @@
 language: cpp
-os:
-- linux
-- osx
-compiler:
-- clang
-- gcc
+
+matrix:
+  include:
+     # Coverage
+     - os: osx
```

Comment:
Any reason this has to be `osx`? Would be quicker to run this as `linux` on `slowtravis` I would think.